internals/types/index.d.ts

```diff
@@ -1,6 +1,11 @@
 import type { BuilderFunctions } from '@vercel/build-utils';
 import type { Readable, Writable } from 'stream';
 import type { Route } from '@vercel/routing-utils';
+import { PROJECT_ENV_TARGET } from '@vercel-internals/constants';
+
+export type ProjectEnvTargetTuple = typeof PROJECT_ENV_TARGET;
```

Comment:
I don't really think we need the Tuple export

Reply:
True I wound up not using it anywhere